### DISCUSSION TITLE
Authentication QR Code

### DISCUSSION THREAD

**Discussion by jamm101** [Tyrannicodin/scuffed-tcg-site#14] [Q&A] [answered]
I signed up by skipping authentication and now that I want to delete my account, the site is asking me for an authentication code. How can I get a new Authentication QR Code?

**Comment by Tyrannicodin** [comment, marked as answer]
You can't. In the next update it will be impossible to not keep a hold of the authentication code.
For now, If you give me the username I will manually delete your account.
